src/awsexplorer/activation.ts

```diff
@@ -197,6 +198,12 @@ async function registerAwsExplorerCommands(
             }
         )
     )
+
+    context.subscriptions.push(
+        vscode.commands.registerCommand('aws.viewLogStream', async (node: LogGroupNode) =>
```

Comment:
Maybe `aws.cloudWatchLogs.viewLogStream`? These commands shouldn't necessarily be exclusive to the explorer (I had a thought at one point to allow this through the command palette, for instance).

Do you think this individual change is one that should be made here or should that wait for the full pass?

Reply:
> These commands shouldn't necessarily be exclusive to the explorer

This command operates on an explorer node (LogGroupNode), it seems likely there would be a similar command for non-explorer situations.

> Do you think this individual change is one that should be made here or should that wait for the full pass?

It would be good to start introducing code that we can use as reference to future implementers instead of propagating patterns we believe should change. I'll defer to your judgement on this one.

Reply:
> > These commands shouldn't necessarily be exclusive to the explorer
> 
> This command operates on an explorer node (LogGroupNode), it seems likely there would be a similar command for non-explorer situations.

The ideal here would be something like:
```
async (node?: LogGroupNode) {
    let logGroup = logGroupNode?.name
    if (!logGroup) {
        logGroup = # workflow to select a log group name
    }
    # continue workflow
}
```

I'd be interested to see if that's a pattern that we'd want going forward so we can have additional ways into each workflow.

> > Do you think this individual change is one that should be made here or should that wait for the full pass?
> 
> It would be good to start introducing code that we can use as reference to future implementers instead of propagating patterns we believe should change. I'll defer to your judgement on this one.

Be the change you want to see in the world, I guess. Coming right up.